tests/lib/rules/object-shorthand.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview Tests for concise-object rule
+ * @author Jamund Ferguson <http://www.jamund.com>
+ * @copyright 2015 Jamund Ferguson. All rights reserved.
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var eslint = require("../../../lib/eslint"),
+    ESLintTester = require("eslint-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var features = {
+    objectLiteralShorthandMethods: true,
+    objectLiteralShorthandProperties: true,
+    arrowFunctions: true,
+    destructuring: true,
+    generators: true
+};
+
+var eslintTester = new ESLintTester(eslint);
+eslintTester.addRuleTest("lib/rules/object-shorthand", {
+    valid: [
+        { code: "var x = {y() {}}", ecmaFeatures: features },
+        { code: "var x = {y}", ecmaFeatures: features },
```

Comment:
You should add some where the property name and value are different, as those should be ignored.